tests/xfstests/run.pm

```diff
@@ -540,10 +540,6 @@ sub run {
         my $whitelist = LTP::WhiteList->new($issues);
         my %skipped = map { $_ => 1 } $whitelist->list_skipped_tests($whitelist_env, $TEST_SUITE);
         %black_list = (%black_list, %skipped);
-        my $known_issues = $whitelist->{whitelist}->{$TEST_SUITE};
```

Comment:
Better to keep 'XFSTESTS_SOFTFAIL', we don't use yaml in developing products like the coming SLM6.1 test. The softfail needs to be added more flexible.

Reply:
My previous comment is invalid, I see you did that in generate_report line 99.